local-cli/runWindows/utils/deploy.js

```diff
@@ -110,14 +110,18 @@ function deployToDesktop(options) {
 
 function startServerInNewWindow(options) {
   return new Promise(resolve => {
-    http.get('http://localhost:8081/status', res => {
-      if (res.statusCode === 200) {
-        console.log(chalk.green('React-Native Server already started'));
-      } else {
-        console.log(chalk.red('React-Native Server not responding'));
-      }
+    if (options.packager) {
```

Comment:
Does this conditional check work as expected? I thought `commander` squashed arguments into camel case? I.e., I would think the check should be:
```
if (!options.noPackager) {
  /* ... */
```